tjmonopix2/scans/scan_ext_trigger.py

```diff
@@ -18,21 +18,19 @@
     'start_row': 0,
     'stop_row': 512,
 
-    'scan_timeout': False,    # Timeout for scan after which the scan will be stopped, in seconds; if False no limit on scan time
```

Comment:
I think we should keep this. One can run with an external trigger, but stop after a given time or after a given number of triggers. Similar to pybar, bdaq53, pymosa etc

Reply:
Done.